digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/process/SpeedLimitValidator.scala

```diff
@@ -1,10 +1,11 @@
 package fi.liikennevirasto.digiroad2.process
 
-import fi.liikennevirasto.digiroad2.asset._
+import fi.liikennevirasto.digiroad2._
+import fi.liikennevirasto.digiroad2.asset.{SideCode, TextPropertyValue, TrafficDirection}
 import fi.liikennevirasto.digiroad2.dao.InaccurateAssetDAO
 import fi.liikennevirasto.digiroad2.dao.pointasset.PersistedTrafficSign
-import fi.liikennevirasto.digiroad2.linearasset.{NumericValue, RoadLink, SpeedLimit}
-import fi.liikennevirasto.digiroad2.service.pointasset.{TrafficSignService, TrafficSignType}
+import fi.liikennevirasto.digiroad2.linearasset.{NumericValue, RoadLink, SpeedLimit => SpeedLimitService}
```

Comment:
This code should reverted
import fi.liikennevirasto.digiroad2.linearasset.{NumericValue, RoadLink, SpeedLimit}